Tools/autotest/quadplane.py

```diff
@@ -1702,7 +1702,7 @@ def ShipLanding(self):
 
         distance = self.get_distance_int(vehicle_gpi, ship_gpi)
         self.progress(f"{distance=}")
-        max_distance = 1
+        max_distance = 1.5
```

Comment:
I think we agreed to your suggestion of 1.2m.

Reply:
> I think we agreed to your suggestion of 1.2m.

OK, changed to 1.2 :-)